apps/docs/content/components/CheckboxGroup/index.mdx

```diff
@@ -0,0 +1,49 @@
+---
+title: CheckboxGroup
```

Comment:
FYI we don't use PascalCase in docs unless we're referring to the React component in the React tab. Should use sentence case.